examples/.config/model_params_pytorch.json

```diff
@@ -161,20 +101,16 @@
     "resnet101_qat": {
       "model_src_dir": "image_recognition/torchvision_models/quantization/qat/eager",
       "dataset_location": "/tf_dataset/pytorch/ImageNet/raw",
-      "input_model": "",
-      "yaml": "conf.yaml",
-      "strategy": "basic",
-      "batch_size": 100,
-      "new_benchmark": false
+      "input_model": "resnext101_32x8d",
```

Comment:
`resnet101` or `resnext101` ？

Reply:
the input model should be `resnext101_32x8d`, so edited the topology name.